simulation/sub8_gazebo/launch/duck.launch

```diff
@@ -2,7 +2,7 @@
   <param name="simulate" value="true" />
```

Comment:
I am having those tf issues too, now. I think it's due to starting out of order.

One reason is that Forrest suggested using sim time even in real operation to force time synchronization